biophysics-and-physicobiology.csl

```diff
@@ -24,10 +24,10 @@
   <macro name="title">
     <choose>
       <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
-        <text variable="title" font-style="italic"/>
+        <text variable="title" text-case="capitalize-first" font-style="italic" suffix="."/>
```

Comment:
Capitalize-first is an odd casing for a title. What is this supposed to fix? Same question below

Reply:
As you may find the style definition and examples in the "Instruction for Editors" (https://www.biophys.jp/biophysics_and_physicobiology03.html), "capitalize-first" is a necessary definition for our Journal, BPPB. Probably there may be a historical reason, and we do not want to change this style at the moment. 
I understand  that text-case="capitalize-first" does not seem to work well in the Zotero system (https://www.zotero.org/), but it works well in the Mendeley Desktop. Although you feel this Capitalized-first is an odd style.

Reply:
What's in your style guide is just sentence case, which is perfectly fine and quite common. What's strange is using CSL capitalize-first for this. In what scenario does this actually produce something different from the current style?

Reply:
I will prepare the results for some examples, later. Please wait for my next reply. Thank you.

Reply:
At the moment, the previous one (#5646) works  well. So, this change may not be necessary.